apps/condo/domains/contact/components/ContactsEditor/ContactOption.tsx

```diff
@@ -20,7 +20,7 @@ export const ContactOption: React.FC<IContactFieldsDisplayProps> = ({ contact, o
                 <PhoneInput
                     disabled
                     value={contact.phone}
-                    style={{ width: '100%', height: '40px' }}
+                    block
```

Comment:
Why not to set default value for `PhoneInput.block` to `true`?
It is being set in every usage place.